lib/game.rb

```diff
@@ -20,33 +20,45 @@ def toggle_current_player
   end
 
   def make_move
-    @display.display_board(@board.board)
+    @display.display_board(@board.squares)
     @display.show_current_player(@current_player.mark)
     move = @display.ask_for_move
-    until move_valid?(move)
-      @display.invalid_move_message(move)
+    until @board.move_valid?(move)
+      @display.show_invalid_move_message(move)
       move = @display.ask_for_move
     end
     @board.player_make_move(@current_player.mark, move.to_i)
   end
 
-  public
+  def winning_player_exists?
```

Comment:
This method calls two methods on `board`, which is a smell that it belongs on `board`.

Reply:
Thanks, Matt. I'll move this method